File: Backend/participant-service/src/main/java/com/ensa/projet/participantservice/dto/TestResultSubmissionDTO.java
package com.ensa.projet.participantservice.dto;

import lombok.Data;
import java.util.List;

@Data
public class TestResultSubmissionDTO {
    private Integer participantId;
    private Integer trainingId;
    private Integer moduleId;
    private List<ParticipantAnswerDTO> answers;
}

File: Mobile/app/src/main/java/com/example/firstaidfront/models/TestResultSubmissionDTO.kt
package com.example.firstaidfront.models

data class TestResultSubmissionDTO(
    val participantId: Int,
    val trainingId: Int,
    val moduleId: Int,
    val answers: List<ParticipantAnswerDTO>
)

// ParticipantAnswerDTO.kt
data class ParticipantAnswerDTO(
    val quizId: Int,
    val selectedAnswerIndex: Int
)